sites/public/src/pages/help/get-started.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useContext } from "react"
```

Comment:
these 3 files are virtually equivalent but I think that DRY-ing them out may be ill advised since they are html pages that will likely diverge as the site evolves, and most of the logic is already deduped within `Layout`